### ISSUE TITLE
运行demo.py出错

### ISSUE THREAD

**Issue by xiaozhi708** [wanghao9610/X-SAM#17]
我将X-SAM-main/xsam/xsam/demo/demo.py放到X-SAM-main/xsam/下，执行reaseg任务，运行后报错：The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's 'attention_mask' to abtain reliable results. 这是因为啥啊，我做reaseg输入就一张图片和一句话吧，这个attention_mask是哪里来的，我该如何修正这个bug。

**Comment by xiaozhi708** [comment]
还有这些报错：The argument 'trust_remote_code' is to be used with Auto classes. It has no effect here and is ignored.
Some weights of SamModel were not initialized from the model checkpoint at ../inits/sam-vit-large and are newly initialized: ['mask_decoder.query_tokens.weight']
You should probably TRAIN this mode on a down-stream task to be able to use it for predictions and inference.
[configuration_mask2former.py:180:__init__] 'backbone_config' is 'None'. Initializing the config with the default 'Swin' backbone.
难道是我下载的sam权重有问题导致的吗？

**Comment by wanghao9610** [comment]
> 我将X-SAM-main/xsam/xsam/demo/demo.py放到X-SAM-main/xsam/下，执行reaseg任务，运行后报错：The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's 'attention_mask' to abtain reliable results. 这是因为啥啊，我做reaseg输入就一张图片和一句话吧，这个attention_mask是哪里来的，我该如何修正这个bug。

这个bug 应该不是按照我提供的demo流程出现的吧？我似乎没遇到过这个error。你确定这是一个warning 不是一个error？

**Comment by xiaozhi708** [comment]
就是普通的打印，没有warning也没有error，但是没有保存预测结果，程序直接结束了。

**Comment by wanghao9610** [comment]
> 因为我在服务器上运行，所有没有用runs/run.sh； 就是普通的打印，没有warning也没有error，但是没有保存预测结果，程序直接结束了。 能否按照我这样运行，复现一下bug，因为没有报错，我都定位不到是哪行代码出的问题。

可以给我一下你执行的命令吗

**Comment by liuyaiai** [comment]
prompt
符合格式的话，默认需要将demo中L579取消注释可以得到可视化结果。
